### ISSUE TITLE
[Build 280] Creepers don't prime before explosion / Explosion doesn't change nearby blocks

### ISSUE THREAD

**Issue by zaptrem** [DragonetMC/DragonProxy#118]
Online mode: False
Server target: Local test server
Client: PE 0.14.0 iOS
Server: Spigot 1.8.3 (Latest before 1.9 builds)

Steps to reproduce:
1. Join server through proxy
2. Wander within Creeper explosion range
3. Notice how it doesn't seem to react until it explodes
4. Creeper disappears, user takes damage, nearby block-items appear on the ground, blocks in nearby area don't change

**Comment by TheDiamondYT1** [comment]
Not implemented yet

**Comment by TheDiamondYT1** [comment]
@DefinitlyEvil Can I advertise DragonetMC is recruiting?

**Comment by DefinitlyEvil** [comment]
@TheDiamondYT1 Yeah, we do need some people develop together.

**Comment by zaptrem** [comment]
If you have any documentation I could take a look around. I haven't
bothered with much Java for quite a while, but it shouldn't be too
different from other object oriented languages, right?

**Comment by DefinitlyEvil** [comment]
@zaptrem Well, there is actually no documentations at all. We use translators to transform the protocol data.

**Comment by TekkitCommando** [comment]
Might be good to think about Dragonet Teams future and get a team management and recruiting system thought up. I can help you with that.

**Comment by TheDiamondYT1** [comment]
I could too... If I get the website code or access to the server where it's hosted.

**Comment by TekkitCommando** [comment]
I own the Dedi it's hosted on but its CPanel and so only one account per domain so you will need to have DefinitlyEvil changed the password to something else and give you access to it too if he wants. Of course there are some guidelines you need to follow before using the panel.

**Comment by zaptrem** [comment]
I just realized the connection method suggested in what little
documentation is provided insists I connect to your auth server without
HTTPS. Are the user details for account holders encrypted on the server?
With salt?

**Comment by TekkitCommando** [comment]
Its encrypted in different ways, but I understand that people don't have access to the server and wouldn't know that so I am working on getting Dragonet and all the services in all of Byte Powered's existence setup with lifetime SSL encryption.

**Comment by TekkitCommando** [comment]
HTTPS For The Win

**Comment by zaptrem** [comment]
What is "Byte Powered?" I had an idea for a proxy similar to this and
decided to search around before actually starting the project. More than a
maintainer and a few contributors, who are you all?

**Comment by zaptrem** [comment]
Free SSL certificates, organization sponsored by many large companies (Mozilla, Google, EFF, etc) https://letsencrypt.org/

**Comment by TekkitCommando** [comment]
Yes LetsEncrypt is what I am setting up on all the severs. I am the owner of Byte Powered, a software development group and soon to be company. Dragonet (before it was abandoned) and DragonProxy are products of Byte Powered along with a CMS software and other things in development and DefinitlyEvil is the lead dev of the Dragonet team and assisted by a bunch of other awesome devs some of which have stayed and others who have contributed a few times and went on their own path. I host the Dragonet website, forums, I don't think we have any right now but I usually do the minecraft server hosting for these guys as well. Its a long process of how all this worked out. Eventually (hopefully) DefinitlyEvil will be coming to the US to work as a full time employee for Byte Powered. We just gotta get our feet off the ground and become net positive first.

**Comment by TekkitCommando** [comment]
Thats my life story as a member of the Dragonet Team :P

**Comment by zaptrem** [comment]
What services do you hope to provide?

**Comment by TheDiamondYT1** [comment]
Dragonet could stay an organization, but be owned by byte powered maybe. I wanna help byte powered.

**Comment by TheDiamondYT1** [comment]
@TekkitCommando ignore everything I just said #Byte4Life

**Comment by TheDiamondYT1** [comment]
xD

**Comment by TekkitCommando** [comment]
@zaptrem We just do software development, most of which open source. @TheDiamondYT1 Dragonet is an organization in its own right, you could consider it funded and put into action by Byte Powered. I think you can tell I leave all authority to DefinitlyEvil and I intend it to stay that way, Byte Powered isn't overlord of this project. You hardly see me talk in the issues for that reason. You are already helping by sticking with this project and thanks I love the hashtag lol

**Comment by TheDiamondYT1** [comment]
And seriously dude.. WordPress!!!!!!

**Comment by TekkitCommando** [comment]
Might be getting a bit off topic

**Comment by TekkitCommando** [comment]
Yeah I love WordPress perfect once you get it right, the site still needs updating though :P

**Comment by TheDiamondYT1** [comment]
WordPress is SHIeT

**Comment by TekkitCommando** [comment]
We are coding our own CMS in PHP and plan to replace the site with it when it's done but until then WordPress it is haha

**Comment by TheDiamondYT1** [comment]
Doesn't @DefinitlyEvil live in the US?

**Comment by TekkitCommando** [comment]
He lives in China, It's tuesday for him :O

**Comment by zaptrem** [comment]
Quite off-topic now. WordPress is a very powerful tool. Why bother writing an entire website/user management system when you can plug into one used by millions and under construction for years? Back when I ran a fairly successful Minecraft server (4 years ago?) WordPress had everything I needed, and I was able to lean on it for user systems whenever I needed to create/integrate anything else.

**Comment by TekkitCommando** [comment]
@zaptrem Because we believe we have developed something unique in the ways of the CMS :P But I agree with you about WordPress being powerful, I love it, I and another person code a WordPress and Spigot/Bukkit plugin called WooMinecraft that allows you to setup a donation store on a WordPress site.

**Comment by TheDiamondYT1** [comment]
Google didn't become huge because it used some random software it found online.

#CustomCoded

**Comment by TekkitCommando** [comment]
Well said

**Comment by zaptrem** [comment]
Google began as a research project. It ordered search results based on links between websites in addition to simple text matching. Nothing like it existed before Larry Page and Sergey Brin created it.

**Comment by TheDiamondYT1** [comment]
I was making a point

**Comment by TheDiamondYT1** [comment]
@TekkitCommndo do you code PHP HTML and js?

**Comment by zaptrem** [comment]
And I, a counter point. Have you ever heard the phrase "Don't reinvent the wheel?" ;)

**Comment by TheDiamondYT1** [comment]
Yes. But you would stand out more if you did.

**Comment by zaptrem** [comment]
Stand out to whom?

**Comment by TekkitCommando** [comment]
What if that wheel was triangle instead of a circle. That's is the mindset lol

**Comment by TheDiamondYT1** [comment]
Idk people who visit the site.

**Comment by TheDiamondYT1** [comment]
Does byte powered just do games?

**Comment by zaptrem** [comment]
For example, why write my own proxy when I can contribute here and use this one?

**Comment by TheDiamondYT1** [comment]
You could make your own.

Like saying "Why make Sponge when there's Spigot"
Or why make yahoo when there's google

**Comment by TekkitCommando** [comment]
Minecraft is the only game we have created or thought about creating software for. There is a plan to make our own game but that's in the future. Minecraft will probably stay the only game we create software for that isn't owned by us. Probably should get back on topic before DefinitlyEvil gets mad at me.

**Comment by zaptrem** [comment]
Yahoo came first, and from what I know Sponge had a good reason to come into being.

**Comment by liach** [comment]
> @TheDiamondYT1
> The Dragonet Team member
> TheDiamondYT1 commented 35 minutes ago
> Doesn't @DefinitlyEvil live in the US?
> @TekkitCommando
>  The Dragonet Team member
> TekkitCommando commented 33 minutes ago
> He lives in China, It's tuesday for him :O

This shocks me...

**Comment by TheDiamondYT1** [comment]
@TekkitCommando Easy money https://mc-market.org.

Sell stuff in resources section

**Comment by zaptrem** [comment]
On topic: How would one go about implementing the Creeper Prime action?

**Comment by zaptrem** [comment]
@TheDiamondYT1 I'm fairly sure the sale of Minecraft mods is against the EULA.

**Comment by TekkitCommando** [comment]
@TheDiamondYT1 good idea. @zaptrem I remember looking at the code for Dragonet a lot not so much for DragonProxy, I assume there's an events folder or something along those lines where you can code a CreeperExplosionEvent I am just spitballing here because I haven't taken the time to get familiar with the classes.

**Comment by TekkitCommando** [comment]
I don't really do mods so I couldn't tell you if it's illegal or not but I doubt it is because it's not mojangs code.

**Comment by TheDiamondYT1** [comment]
Selling mods is against, and might be for plugins, but

A) sell websites, configs etc instead
B) deal with it. Everyone breaks the EULA.

**Comment by TekkitCommando** [comment]
Lol, it is legal to sell almost everything else though so still some money to made probably.

**Comment by TheDiamondYT1** [comment]
Sombeody sold a config for $2 something today it's got 63 downloads.

**Comment by TheDiamondYT1** [comment]
And you can pay me by buying me premium on MC mrket Lmao

**Comment by TekkitCommando** [comment]
Lol nice and hmm let me think about that xD

**Comment by zaptrem** [comment]
I remember two events happening as I drifted away from all of this. First, I remember the mass panic of the server owners as they tried to figure out ways to stay afloat while complying with the EULA. Second, I remember Bukkit shutting down for whatever reason and nearly bringing Spigot with it, while the major developers in the community jumped ship to a new project, Sponge (how is it doing?). Has much else happened since then? -- Why would anyone buy anything on that store when they can find a free version on BukkitDev or the Spigot equivalent?

**Comment by TekkitCommando** [comment]
Cuz it's usually websites and scripts people want from that site and sponge is doing fine it's in beta now.

**Comment by zaptrem** [comment]
Wow, just a look at all these old (and new!) names over at spongepowered.org makes me want to jump back in to this community.

**Comment by TheDiamondYT1** [comment]
@tekkitcommando skype name?

**Comment by TekkitCommando** [comment]
@TheDiamondYT1 ethan.smith184
